packages/valory/skills/decision_maker_abci/policy.py

```diff
@@ -89,6 +89,7 @@ class EGreedyPolicy:
     eps: float
     accuracy_store: Dict[str, AccuracyInfo] = field(default_factory=dict)
     weighted_accuracy: Dict[str, float] = field(default_factory=dict)
+    updated_ts: int = 0
```

Comment:
Maybe this fits better in the accuracy store, which is what gets updated from the global info.

Reply:
Added updated_ts here as it needs to be written to the policy store as a log of the last time of updating the  local store. The policy store is updated by writing the policy to the store.

Reply:
The `accuracy_store` belongs to the policy, therefore it will still be written.